src/components/box/README.md

```diff
@@ -1 +1,241 @@
+
+```
+<div class="rvt-box rvt-box--compact">
+  <div class="rvt-box__header">
+    A box with rows
+  </div>
+  <ul class="rvt-plain-list">
+    <li class="rvt-box__row">
+      Some text in a row
+    </li>
+    <li class="rvt-box__row">
+      Some text in a row
+    </li>
+    <li class="rvt-box__row">
+      Some text in a row
+    </li>
+  </ul>
+</div>
+```
+
+## Colored box
+
+The box component also supports colored variants based on [Rivet’s alert color naming conventions](#).
+
+The colored box variants should be used sparingly and only to indicate some sort of state (error, warning, success, etc.) or call out important information in your application.
+
+There are four colored box variants that follow Rivet’s alert color naming convention:
+
+- `.rvt-box--info`
+- `.rvt-box--success`
+- `.rvt-box--warning`
+- `.rvt-box--danger`
+
+```
+<div class="rvt-box rvt-box--danger">
+  <div class="rvt-box__header">
+    A box with rows
+  </div>
+  <ul class="rvt-plain-list">
+    <li class="rvt-box__row">
+      Some text in a row
+    </li>
+    <li class="rvt-box__row">
+      Some text in a row
+    </li>
+    <li class="rvt-box__row">
+      Some text in a row
+    </li>
+  </ul>
+</div>
+```
+
+### Colored box row variants
+
+Colors can also be applied to individual rows to call out state changes or important information:
+
+- `.rvt-box__row--info`
+- `.rvt-box__row--success`
+- `.rvt-box__row--warning`
+- `.rvt-box__row--danger`
+
+The row variant modifier classes add a subtle background color and left border that matches each of Rivet’s alert styles.
+
+```
+<div class="rvt-box">
+  <div class="rvt-box__header">
+    A box with rows
+  </div>
+  <div>
+    <div class="rvt-box__row">
+      Default row
+    </div>
+    <div class="rvt-box__row rvt-box__row--info">
+      Info row variant
+    </div>
+    <div class="rvt-box__row rvt-box__row--success">
+      Success row variant
+    </div>
+    <div class="rvt-box__row rvt-box__row--warning">
+      Warning row variant
+    </div>
+    <div class="rvt-box__row rvt-box__row--danger">
+      Danger row variant
+    </div>
+  </div>
+</div>
+```
+
+## Card variant
+
+The card variant of the box component can be used along with Rivet’s [type scale utilities](#) and [spacing utilities](#) to create basic card layouts. Adding the `rvt-box--card` modifier class adds a subtle shadow to help visually separate the the box content from the background.
+
+```
+<div class="rvt-box rvt-box--card">
+  <div class="rvt-box__body">
+    This is the card variant of a box with a body element
+  </div>
+</div>
+```
+
+### Card image
+
+The card variant can include an optional image. Nesting an `img` inside an element with the `rvt-box__image` class will apply the proper sizing, cropping, and border radius.
```

Comment:
```suggestion
The card variant can include an optional image. Nesting an `img` inside an element with the `rvt-box__image` class will apply the proper sizing, cropping, and border radius.

**Card images must be the first child** The element with the `rvt-box__image` class must be the first child of `.rvt-box` so that the image sits flush with the sides of the box (see above example). It should not be used inside elements with the `rvt-box__body` or `rvt-box__row` classes.
```